orttraining/orttraining/python/training/onnxblock/building_blocks.py

```diff
@@ -178,3 +179,19 @@ def __init__(self):
 class Neg(_UnaryOp):
     def __init__(self):
         super(Neg, self).__init__("Neg")
+
+
+class Value(Block):
```

Comment:
in which case do we need this? just for during some loss function building?

Looks they will not be trainable param, right? so why we call it weight?

Reply:
One of the user model has two outputs. The training model is built by combining these two outputs as weighted average:
```py
output = w1*celoss(output0)+w2*celoss(output1)
```

For the user to be able to do this, we need a way for them to define what w1 and w2 is. Calling it weight only in the unit tests which uses the initializers as weighted average coefficients.